input/ig.xml

```diff
@@ -246,31 +246,31 @@
 				<reference value="Condition/ckd"/>
 			</reference>
 			<name value="Condition - Chronic Kidney Disease"/>
-			<description value="Chronic Kidney Disease &#xa;&#xa;Patient: Mr. Harry Dan"/>
+			<description value="Shows an example of an active condition for a patient with an onset age for the *AU Core Condition* profile.&#xa;&#xa;Patient: Mr. Harry Dan"/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-condition"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Condition/uti"/>
 			</reference>
 			<name value="Condition - Urinary tract infection"/>
-			<description value="Urinary tract infection&#xa;&#xa;Patient: Mia LEANNE BANKS"/>
+			<description value="Shows an example of a resolved condition for the *AU Core Condition* profile.&#xa;&#xa;Patient: Mia LEANNE BANKS"/>
```

Comment:
Missing info on the condition itself, and the other things the example demonstrates. Above says onset age, this has both onset and abatement.

Reply:
Updated